react_redux/README.md

```diff
@@ -585,7 +563,7 @@
     export default Link;
     ```
 
```

Comment:
The thing is in the above code, it says it is an example, but the Typescript way is very different than this. Also use Typescript in each of our project. It'd be weird to say things like "You forgot to type your variable here." when nothing is typed in the examples.